GitHub-Test-DHprojects/Gardiner_GitTest4.md

```diff
@@ -0,0 +1,9 @@
+# Joey Gardiner Git Test 4
+
+This project team originally sought to find what types of `obsolete` words were used in Lovecraft's writing. So the team shifted their focus to finding what words Lovecraft used to elicit a feeling of **confusion, ambiguity, and dread.** They measured the terms by categories of `scientific, religious, gothic, or other`.  
+
+
+I thought this was a very thoughtful way to go about searching for what about Lovecraft's writing evokes **uneasiness**. This project helps pinpoint what makes Lovecraft's writing to interesting and well-written. It can help writers that want to create a piece with the type of cosmic or psychological horror Lovecraft writes, take a deeper look into the text and really find what provokes such feelings so they can *utilize similar ideas*.
```

Comment:
I myself have never read Lovecraft, but from excerpts it's immediately obvious that he has a very unique style that lends itself well to interesting psychological horror.